module.go

```diff
@@ -38,6 +35,11 @@ type ScanResponse struct {
 	Error     *string     `json:"error,omitempty"`
 }
 
+type ScanModuleAttributes struct {
+	Name        string
+	DefaultPort uint
```

Comment:
Is `DefaultPort` intended for use in a future PR to replace the hard-coded default in the modules' calls to `zgrab2.AddCommand`?

Reply:
My only concern is if `DefaultPort` goes into this PR, and subsequent PRs that use it are not merged, it becomes confusing where to change / add the default port for existing / new scanners.  I'd recommend either pulling `DefaultPort` out, or go ahead and use it in the `AddCommand` calls.